protostar/commands/build/build_command.py

```diff
@@ -47,6 +47,6 @@ def arguments(self) -> List[Command.Argument]:
     async def run(self, args):
         self._project_compiler.compile(
             output_dir=args.output,
-            cairo_path=args.cairo_path,
+            extra_cairo_paths=args.cairo_path,
```

Comment:
It's not consistent with array naming convention in Python. Python domain => Python rules.

@mkaput WDYT?

Reply:
I am 50/50, about this.

If we want to follow the list naming convention I wouldn't use name `cairo-path` to avoid naming conflict. 

Maybe  the best would be to stick with `include_paths` for internals and expose `cairo-path` to the user?

Reply:
`include` is a verb. It shouldn't be used for a variable name.

Reply:
`paths_to_include`, `included_paths`

Reply:
`abs_cairo_paths` and `rel_cario_paths`?

Reply:
My opinion is to:
1. stick with naming used in Cairo nomenclature, which I guess is `cairo_path`
2. canonicalize absolute/relative paths to common form as early as possible